setup.cfg

```diff
@@ -2,7 +2,7 @@
 name = plasmapy
 provides = plasmapy
 # version should be PEP440 compatible (https://www.python.org/dev/peps/pep-0440/)
-version = 0.3.0.dev
+version = 0.4.0.dev
```

Comment:
Do we need to increment to 0.4.0.dev in this PR yet?

Reply:
I would say this should be ``0.3.0`` until after the release.

Reply:
It should have been, I forgot to set it to 0.3.0 for the release, and now I'm confused on how best to proceed. Perhaps let's just get some neat feature out and get 0.4.0 out the door :sweat_smile:

Reply:
Actually, while I did set it up incorrectly, I actually think it might have gone okay if setuptools_scm pulled it from the git tag. I'm having trouble reproducing it, though.

Reply:
And since this is now done by setuptools_scm, we don't need this field in the cfg file at all. I've removed it in the current version.